src/pages/_app.tsx

```diff
@@ -3,6 +3,11 @@ import '../styles/global.scss';
 import type { ReactNode } from 'react';
 import type { NextPage } from 'next';
 import type { AppProps } from 'next/app';
+import { Roboto } from '@next/font/google';
+
+const roboto = Roboto({
+    weight: ['100', '400', '700'],
```

Comment:
We probably won't be using `100`.
```suggestion
    weight: ['400', '700'],
```